### ISSUE TITLE
Add option to wait for zpool scrub/run zpool scrub in foreground until completion

### ISSUE THREAD

**Issue by krichter722** [zfsonlinux/zfs#6836]
### Describe the problem you're observing
There's no way to wait for `zpool scrub` by running the process in the foreground and make it return only after the scrub has been completed. This makes the use of the command unnecessarily difficult in maintenance scripts (e.g. cron scripts) for which one might want to generate a result email for and send it.

I know about ways to achieve that with zed or parse the output of `zpool status`, but I consider them workarounds for this easier and more intuitive approach.

**Comment by kingneutron** [comment]
> I know about ways to achieve that ... parse the output of zpool status, but I consider them workarounds for this easier and more intuitive approach

--In the meantime, for anyone who could use an easy workaround, this should work in bash while a scrub is running:
`
# do forever until scrub is done
while :; do
  zpool status |tee -a ~/scrublog.log |grep -A 2 'scrub in progress' || break 2
  sleep 30
done
`

--The script attached can be run interactively, and also shows pool I/O during the scrub. HTH
[scrubwatch.tar.gz](https://github.com/zfsonlinux/zfs/files/1516111/scrubwatch.tar.gz)

**Comment by baryluk** [comment]
What about `zpool scrub -w`? 

From the manpage:

```
             -w      Wait until scrub has completed before returning.
```

**Comment by behlendorf** [comment]
The `zpool scrub -w` option and `zpool wait` subcommand were only recently added.  They will be included in the OpenZFS 2.0 release, but are not part of the 0.8.0 releases.  However, since this has been implemented and merged I'm going to go ahead and close this issue.
